tests/TestCase/ORM/MarshallerTest.php

```diff
@@ -2745,7 +2795,7 @@ public function testMergeJoinDataWithFieldList()
 
         $tag1 = $entity->tags[0];
         $result = $marshall->merge($entity, $data, [
-            'associated' => ['Tags._joinData' => ['fieldList' => ['foo']]]
+            'associated' => ['Tags._joinData' => [$fields => ['foo']]]
```

Comment:
Could we keep the old and new test alongside and mark the old tests one deprecated (with version)?